src/components/HookForm/index.jsx

```diff
@@ -508,61 +538,76 @@ export default class HookForm extends Component {
               />
             </FormGroup>
           </ListItem>
+          <ListItem className={classes.displayBlock}>
+            <ListItemText
+              primary={
+                <Typography variant="subtitle1">
+                  Last Hook Fire Attempts
+                </Typography>
+              }
+            />
+            {hookLastFires && (
+              <DataTable
+                items={hookLastFires}
+                headers={['Task ID', 'FiredBy', 'Result', 'Attempted', 'Error']}
+                isPaginate
+                renderRow={hookFire => (
+                  <TableRow key={hookFire.taskId}>
+                    <TableCell>
+                      {(hookFire.result === 'SUCCESS' && (
+                        <Link to={`/tasks/${hookFire.taskId}`}>
+                          <Typography>{hookFire.taskId}</Typography>
+                        </Link>
+                      )) || <Typography>{hookFire.taskId}</Typography>}
+                    </TableCell>
+                    <TableCell>
+                      <Typography>{hookFire.firedBy}</Typography>
+                    </TableCell>
+                    <TableCell>
+                      <Label mini status={hookFire.result.toLowerCase()}>
+                        {hookFire.result}
+                      </Label>
+                    </TableCell>
+                    <TableCell>
+                      <DateDistance from={hookFire.taskCreateTime} />
+                    </TableCell>
+                    <TableCell className={classes.errorTableCell}>
+                      {(hookFire.result === 'ERROR' && (
+                        <ExpansionPanel>
```

Comment:
Is there a hook I can see where `hookFire.result === 'ERROR'`? I want to see how it looks in the UI.